forge-gui/res/cardsfolder/upcoming/ace_fearless_rebel.txt

```diff
@@ -0,0 +1,12 @@
+Name:Ace, Fearless Rebel
+ManaCost:3 G
+Types:Legendary Creature Human Rebel
+PT:2/2
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPutCounter | OptionalDecider$ You | TriggerDescription$ Nitro-9 — Whenever CARDNAME attacks, you may sacrifice an artifact. When you do, put a +1/+1 counter on CARDNAME, then it fights up to one target creature defending player controls.
```

Comment:
This ability is currently broken, and I'm not quite sure why.

Reply:
Use AB$ instead of DB$ in line 6. Otherwise the cost will be skipped

Reply:
Also, the oracle says "when you do" which means you will have to use an immediatetrigger

Reply:
Thank you.